dnf/cli/commands/__init__.py

```diff
@@ -114,8 +114,7 @@ def base(self):
         return self.cli.base
 
     @property
-    def basecmd(self):
-        # :api
```

Comment:
@mluscon It is not documented as API, therefore I try to close the gap between documentation and code by romewing of API label and mark it as private